src/docker-compose.yml

```diff
@@ -1,14 +1,12 @@
 services:
   rasa:
-    image: rasa-src-ro
+    image: ancadavid/chatbot-polly:rasa-src-ro
```

Comment:
Can the path be renamed? Having someone's name here is not ideal.

Reply:
This is my username on the docker hub and it's needed to identift the repo where the images are. As we discussed, I will create an account on the docker hub with an email associated with the app. I will use the new username on the first occasion when I have to train the models again (so I will build the images as well)